### PR TITLE
Update Chinese and Japanese localization files (24.1)

### PR DESCRIPTION
**Pull request by nickshulman**
Update resx files with localization.zh-CHS.csv and localization.ja.csv that we got back from translators. The commands that I executed were:
```
pwiz_tools\Skyline\Executables\DevTools\ResourcesOrganizer\scripts\readResxFiles.bat
pwiz_tools\Skyline\Executables\DevTools\ResourcesOrganizer\scripts\exe\ResourcesOrganizer.exe importLocalizationCsv
pwiz_tools\Skyline\Executables\DevTools\ResourcesOrganizer\scripts\exe\ResourcesOrganizer.exe exportResx ResxFiles.zip
and then unzip "ResxFiles.zip"
```

[localization.zh-CHS.csv](https://github.com/user-attachments/files/17298908/localization.zh-CHS.csv)
[localization.ja.csv](https://github.com/user-attachments/files/17330753/localization.ja.csv)

### CODE REVIEW

**Review by brendanx67 — CHANGES_REQUESTED** [review]
You have some punctuation-only changes that are changing from Chinese colons and elipses to Roman punctuation. These changes need to be reverted.

**Comment by nickshulman** [comment]
> You have some punctuation-only changes that are changing from Chinese colons and ellipses to Roman punctuation. These changes need to be reverted.

Fixed. It sounds like the .csv files that were sent to the localizers were generated before the July 14th commit "normalize punctuation use across translations".

File: pwiz_tools/Skyline/FileUI/PeptideSearch/SearchSettingsControl.zh-CHS.resx [hunk 1]
```diff
@@ -780,7 +780,7 @@
     <value>2</value>
   </data>
   <data name="lblHardklorSignalToNoise.Text" xml:space="preserve">
-    <value>最小信噪比(&amp;S)：</value>
+    <value>最小信噪比(&amp;S):</value>
```
Comment (brendanx67) [inline pwiz_tools/Skyline/FileUI/PeptideSearch/SearchSettingsControl.zh-CHS.resx:783]: Chinese to Roman punctuation change

File: pwiz_tools/Skyline/Properties/Resources.zh-CHS.resx [hunk 2]
```diff
@@ -3995,11 +3995,11 @@
     <value>..\Resources\koina-logo.fdf731d5.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
   <data name="CommandLine_ImportPeptideList_Importing_peptide_list__0__from_file__1____" xml:space="preserve">
-    <value>正在导入文件 {1} 中的肽段列表 {0}…</value>
+    <value>正在导入文件 {1} 中的肽段列表 {0}...</value>
```
Comment (brendanx67) [inline pwiz_tools/Skyline/Properties/Resources.zh-CHS.resx:3998]: Chinese to Roman punctuation change

File: pwiz_tools/Skyline/Properties/Resources.zh-CHS.resx [hunk 3]
```diff
@@ -3995,11 +3995,11 @@
     <value>..\Resources\koina-logo.fdf731d5.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
   <data name="CommandLine_ImportPeptideList_Importing_peptide_list__0__from_file__1____" xml:space="preserve">
-    <value>正在导入文件 {1} 中的肽段列表 {0}…</value>
+    <value>正在导入文件 {1} 中的肽段列表 {0}...</value>
     <comment>Needs Review:New resource</comment>
   </data>
   <data name="CommandLine_ImportPeptideList_Importing_peptide_lists_from_file__0____" xml:space="preserve">
-    <value>正在导入文件 {0} 中的肽段列表…</value>
+    <value>正在导入文件 {0} 中的肽段列表...</value>
```
Comment (brendanx67) [inline pwiz_tools/Skyline/Properties/Resources.zh-CHS.resx:4002]: Chinese to Roman punctuation change

File: pwiz_tools/Skyline/Skyline.zh-CHS.resx [hunk 4]
```diff
@@ -1462,7 +1462,7 @@ Old Localized:Show Mass Error</comment>
     <value>194, 22</value>
   </data>
   <data name="importFeatureDetectionMenuItem.Text" xml:space="preserve">
-    <value>特征检测(&amp;F)…</value>
+    <value>特征检测(&amp;F)...</value>
```
Comment (brendanx67) [inline pwiz_tools/Skyline/Skyline.zh-CHS.resx:1465]: Chinese to Roman punctuation change

**Review by brendanx67 — CHANGES_REQUESTED** [review]
I have highlighted 4 remaining cases.

File: pwiz_tools/Skyline/Properties/Resources.zh-CHS.resx (hunk 2 above)
Comment (chambm) [reply pwiz_tools/Skyline/Properties/Resources.zh-CHS.resx:3998]: Knowing enough Chinese to embarrass myself talking to children, I was curious to learn about the Chinese version of the ellipsis. Apparently it's formally six dots, usually two of those 3-dot characters in a row. https://www.zdic.net/ts/fulu/2019/05/27.html
But if the translators were just using one of the 3-dot characters, obviously go with that. :P It's more consistent with the English version that way.

**Review by brendanx67 — APPROVED** [review]
Looks good.

**Pull request merged**